s3prl/downstream/asr/config.yaml

```diff
@@ -34,17 +34,17 @@ specaug:
 
 downstream_expert:
   datarc:
-    train: ['train-clean-100']
-    dev-clean: ['dev-clean']
-    dev-other: ['dev-other']
-    test-clean: ['test-clean']
-    test-other: ['test-other']
+    train: ['train']
```

Comment:
did you download librispeech yourself and then untar the files? i think by default you get folders like `train-clean-100` so you shouldn't need to change these values in the config

Reply:
No, I used the command shared in the PR description (using datasets), which generated a data directory but the CSV names don't have clean on them

Reply:
ah i see the problem and indeed we should probably remove the `datasets` usage since we've now decided to simply mount an external volume for each training job.

originally i thought we'd be able to integrate directly with `datasets` but this turned out to be a nightmare with the way `s3prl` is designed. 

i'll take debug your branch and let you know when it's up and running!

Reply:
Ah ok, I was confused since I was just following the documentation :) It would be great if we could remove all of obsolete stuff.